scripts/validate_systs.py

```diff
@@ -0,0 +1,382 @@
+"""
+  Date:     200325
```

Comment:
I think it would be better to have docs like this be in the README or some other Markdown file for the repo, rather than at the top of the code, given that this is meant to be a utility.

Reply:
This could get taken care of later in a docs PR though.

Reply:
I'm happy to take care of it now as well :)

Reply:
Ah, I see it was just approved - ok, I'll do a separate docs PR!

Reply:
Either is good. I just know that you've waited long enough for this to get in and moving this to a Markdown doc isn't really a good reason for me to hold this up. :)